pilot_urjc_bringup/maps/sim/tb3_world/map.yaml

```diff
@@ -0,0 +1,7 @@
+image: /home/mario/map.pgm
```

Comment:
This image path probably is not valid

Reply:
I'll change that - copy/paste error I guess

Reply:
Solved in 062f8f9ac25701a33de5b9836094f1808d559c49